content/integrations/memcached/Go.md

```diff
@@ -210,10 +210,8 @@ func enableOpenCensusTracingAndMetrics() (func(), error) {
 	// Register as a tracing exporter.
 	trace.RegisterExporter(sd)
 
-	// Register as a metrics exporter.
-	view.RegisterExporter(sd)
-
-	view.SetReportingPeriod(60 * time.Second)
```

Comment:
Update the `Options` as well? https://github.com/census-instrumentation/opencensus-website/pull/710/files#diff-a3d3619a8096fe28463782395db182e0L201

Reply:
I updated the option in Stackdriver.md but left this one as is. The reason is 60s is default and we don't any more configuration than necessary.